manager/integration/tests/test_ha.py

```diff
@@ -139,6 +139,12 @@ def ha_rebuild_replica_test(client, volname):   # NOQA
 def test_ha_salvage(client, core_api, volume_name, disable_auto_salvage):  # NOQA
     """
     [HA] Test salvage when volume faulted
+    TODO
+    The test cases should cover the following four cases:
+    1. Manual salvage with revision counter enabled.
+    2. Manual salvage with revision counter disabled.
```

Comment:
The test step for this case is missed.

Reply:
@shuo-wu , that's the current test implementation. I was expecting whoever works on this test case will restructure it. Thanks.